### PR TITLE
Issue 3 - User can edit an entry

### PR DESCRIPTION
**Pull request by silva-tim**
Video sent to Shawn on Slack.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,6 +20,7 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <script src="https://kit.fontawesome.com/9081f0df74.js" crossorigin="anonymous"></script>
```
Comment (shawnkost) [inline index.html]: I don't believe this is needed? I believe we already imported font awesome for you

File: js/main.js [hunk 2]
```diff
@@ -93,10 +144,35 @@ function viewSwap(view) {
...
 $aNew.addEventListener('click', function (event) {
   viewSwap('entry-form');
 });
+
+// Event to setup form for the editing ability. Inputs all values of data.editing and switches view to entry-form.
+$ul.addEventListener('click', function (event) {
+  if (event.target.tagName !== 'I') {
+    return;
+  }
+  viewSwap('entry-form');
```
Comment (shawnkost) [inline js/main.js]: I would move this to the end of the function, so we swap after we fill in all the inputs. With this approach, we might swap views quickly and notice all the `input`s are blank for a second or so

**Review by shawnkost — APPROVED** [review]
Nicely done Tim! Left a couple comments here

File: index.html (hunk 1 above)
Comment (silva-tim) [reply index.html]: Checked and it is not needed, thanks!

**Pull request merged**